Mathlib/MeasureTheory/Constructions/Convolution.lean

```diff
@@ -0,0 +1,101 @@
+/-
+Copyright (c) 2023 Josha Dekker. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Josha Dekker
+-/
+import Mathlib.MeasureTheory.Constructions.Prod.Basic
+import Mathlib.MeasureTheory.Measure.MeasureSpace
+/-!
+# The convolution of measures
+
+In this file we define and prove properties about the convolution of two measures
+
+## Main definition
+
+* `MeasureTheory.Measure.conv`: The convolution of two measures.
+
+## Main results
+
+* add
+## Implementation Notes
+
+## Tags
+
+-/
+
+noncomputable section
+
+namespace MeasureTheory
+
+namespace Measure
+
+/-- Convolutions of measures. They are defined for arbitrary measures on a monoid M that is also
+a measurable space. TODO: should get a to_additive version for AddMonoids -/
+def conv {M : Type*} [Monoid M] [MeasurableSpace M] (μ : Measure M) (ν : Measure M) :
+    Measure M := Measure.map (fun x : M × M ↦ x.1 * x.2) (Measure.prod μ ν)
```

Comment:
When talking about convolution of measures, one usually refers to the additive version. Could you give another name to the multiplicative definition, use to_additive to get an additive convolution, and name that one `conv`?

Reply:
@RemyDegenne, do you have a good suggestion? An obvious candidate would be `mul_conv`, but that could also refer to something about multiplying and taking convolutions...

Reply:
The name should not have _ in it. It would rather be something like `mulConv`. The multiplicative version of `abs` is `mabs`. Let's use `mconv`?